extensions/notebook-renderers/src/index.ts

```diff
@@ -198,13 +198,25 @@ function onScrollHandler(e: globalThis.Event) {
 	}
 }
 
+function onKeypressHandler(e: KeyboardEvent) {
+	if (e.ctrlKey || e.shiftKey) {
+		return;
+	}
+	if (e.code === 'ArrowDown' || e.code === 'End' || e.code === 'ArrowUp' || e.code === 'Home') {
+		// These should change the scroll position, not adjust the selected cell in the notebook
+		e.stopPropagation();
```

Comment:
It works with `tabindex=0` set. I found that if the scrollable output already scrolls to the bottom, `ArrowDown` will not do anything (since it always `stopPropagation`), I was expecting it to move to the next cell. Do we want to do the same checks as we did for `scroll` (propagate if it's already at the bottom) https://github.com/microsoft/vscode/blob/d4a3b41341d12e7b5a6e55439738d1ae65e4a269/src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts#L403-L434

Reply:
I wouldn't think so, accidentally navigating away from the output when you are trying to scroll would be pretty disorienting, and I imagine not great for accessibility.